src/elements/content-sidebar/activity-feed/annotations/AnnotationActivityLink.tsx

```diff
@@ -1,23 +1,34 @@
 import * as React from 'react';
+import classNames from 'classnames';
 import noop from 'lodash/noop';
-import { FormattedMessage, MessageDescriptor } from 'react-intl';
+import { injectIntl, MessageDescriptor, WrappedComponentProps } from 'react-intl';
 import PlainButton from '../../../../components/plain-button';
 import { ButtonType } from '../../../../components/button';
+import './AnnotationActivityLink.scss';
 
-export interface AnnotationActivityLinkProps {
+type MessageDescriptorWithValues = {
+    values?: Record<string, number>;
+} & MessageDescriptor;
+export interface AnnotationActivityLinkProps extends WrappedComponentProps {
+    className?: string;
     id: string;
     isDisabled: boolean;
-    message: MessageDescriptor;
+    message: MessageDescriptorWithValues;
     onClick: (id: string) => void;
 }
 
 const AnnotationActivityLink = ({
+    className,
     id,
+    intl,
     isDisabled = false,
     message,
     onClick = noop,
     ...rest
 }: AnnotationActivityLinkProps): JSX.Element => {
+    const { values, ...messageDescriptor } = message;
```

Comment:
Could we simplify this by passing in `isFileVersionUnavailable` and manage the message in here, instead?